test/tests/diff.js

```diff
@@ -229,7 +229,7 @@ describe("Diff", function() {
         return diff.patches();
       })
       .then(function(patches) {
-        assert.equal(patches.length, 84);
+        assert.ok(patches.length === 84 || patches.length === 85);
```

Comment:
Can you add a comment here? Future us is going to look at this and be like "what the hell is happening?? Lets fix this". Add a link to #746 in the comment as well please.